docs/app/demo/typography/typography-demo.component.html

```diff
@@ -0,0 +1,422 @@
+            300
+          </li>
+          <li>
+            <span class="type-label">Color</span>
+            $text - $offblack
+          </li>
+        </ul>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <p class="ignore-demo">p. Paragraph</p>
+      </td>
+      <td>
+        <code>&lt;p&gt;&lt;/p&gt;</code>
+        <span class="type-label">or</span>
+        <code>&lt;div class=&quot;p&quot;&gt;&lt;/div&gt;</code>
+      </td>
+      <td>
+        <ul>
+          <li>
+            <span class="type-label">Font-size</span>
+            1.143rem or 16px
+          </li>
+          <li>
+            <span class="type-label">Line Height</span>
+            1.25em
+          </li>
+        </ul>
+      </td>
+      <td>
+        <ul>
+          <li>
+            <span class="type-label">Weight</span>
+            300
+          </li>
+          <li>
+            <span class="type-label">Color</span>
+            $text - $offblack
+          </li>
+        </ul>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <a href="#">a. Anchor</a>
+      </td>
+      <td>
+        <code>&lt;a href=&quot;...&quot;&gt;&lt;/a&gt;</code>
+      </td>
+      <td>
+        <ul>
+          <li>
+            <span class="type-label">Font-size</span>
+            inherit
+          </li>
+          <li>
+            <span class="type-label">Line Height</span>
+            inherit
+          </li>
+        </ul>
+      </td>
+      <td>
+        <ul>
+          <li>
+            <span class="type-label">Weight</span>
+            inherit
+          </li>
+          <li>
+            <span class="type-label">Color</span>
+            $primary-brand - $blue
+          </li>
+        </ul>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <em>em. Emphasised</em>
+      </td>
+      <td>
+        <code>&lt;em&gt;&lt;/em&gt;</code>
+      </td>
+      <td>
+        <ul>
+          <li>
+            <span class="type-label">Font-size</span>
+            inherit
+          </li>
+          <li>
+            <span class="type-label">Line Height</span>
+            inherit
+          </li>
+        </ul>
+      </td>
+      <td>
+        <ul>
+          <li>
+            <span class="type-label">Weight</span>
+            inherit
+          </li>
+          <li>
+            <span class="type-label">Color</span>
+            $gray-500
+          </li>
+        </ul>
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <strong>strong. Strong importance</strong>
+      </td>
+      <td>
+        <code>&lt;code&gt;&lt;/code&gt;</code>
+      </td>
+      <td>
+        <ul>
+          <li>
+            <span class="type-label">Font-size</span>
+            inherit
+          </li>
+          <li>
+            <span class="type-label">Line Height</span>
+            inherit
+          </li>
+        </ul>
+      </td>
+      <td>
+        <ul>
+          <li>
+            <span class="type-label">Weight</span>
+            700
```

Comment:
I've found that font-weight: 700 is really heavy.  Can you change this to 600 (and in the typography scss too)?  I'm probably going against the style guide in places here, but that's the beauty of building this library - to actually see what these values look like in context.